src/hooks/api/discoverSellerSignature.ts

```diff
@@ -23,14 +26,19 @@ export const useDiscoverSellerSignature = (
       console.log('useDiscoverSellerSignature');
       console.log({ discoverSellerSignatureParams: params });
 
+      console.log({ sellerSigParams: params });
```

Comment:
logging the same thing twice? Looks like this file could use some cleanup on logs